CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## To be released
 - Include pushclient module in android build
+- Include PushClient module in iOS build
```

Comment:
@jeremywiebe this is the casing used by the Xcode project..they are both matching their platform styles :/